.github/workflows/actions.yml

```diff
@@ -18,6 +18,7 @@ on:
       - 'companion/**'
       - '.gitpod.yml'
       - '.devcontainer/**'
+      - 'CMakeUserPresets.json'
```

Comment:
@pfeerick did you actually mean `CMakePresets.json`? `CMakeUserPresets.json` is a local file and should never land in our repo (which is why it's in `.gitignore`).

Reply:
Yeah, I copied the name you documented rather than the file checked in. 🤦 I had in mind the classic "check in then ignore" use of `.gitignore` before `git update-index --skip-worktree` and `git update-index --assume-unchanged` were a thing.